### ISSUE TITLE
RSPdx support

### ISSUE THREAD

**Issue by AsciiWolf** [ericek111/libmirisdr-5#6]
Please, consider adding support for SDRplay RSPdx. Thanks!

**Comment by fizr5** [comment]
And do not forget the RSPdx-R2 https://www.sdrplay.com/resources/RSPdxR2Datasheet.pdf  🏭

**Comment by Peter3579** [comment]
The SDRplay hard- and software are closed source. Supporting these receivers would require reverse engineering and access to the chip datasheets.

**Comment by AsciiWolf** [comment]
True. I can at least offer a bounty of 120 USD if someone manages to get the RSPdx working - at least a very basic support without advanced things/register settings like IF/RF gain, Bias-T control etc. These can be added later.

The original SDRplay host libraries are awful. Truly awful. I love the hardware, but its proprietary "API drivers" are probably the worst thing I have **ever** seen (and installed) on my Linux desktop. And there is no way to get them working with Flatpak/Snap SDR applications.
